pkg/sources/adapter/webhooksource/webhook.go

```diff
@@ -70,7 +70,7 @@ func (h *webhookHandler) Start(ctx context.Context) error {
 	return runHandler(ctx, s)
 }
 
-// runHandler runs the HTTP event handler until ctx get cancelled.
+// runHandler runs the HTTP event handler until ctx gets cancelled.
```

Comment:
please excuse me for adding this change. Was using this PR to debug the CI build